java/core/src/test/java/com/redhat/rhn/taskomatic/core/test/TestDaemon.java

```diff
@@ -81,6 +81,7 @@ public void run() {
                 Thread.sleep(10000);
             }
             catch (InterruptedException e) {
+                //test should never be here
```

Comment:
Since this is unexpected, I guess resetting the interrupt flag in this case could be the best choice:

```suggestion
                Thread.currentThread().interrupt();
```

Reply:
fair enough, fixed